### PR TITLE
Add Apihouse, an alternative site of Public APIs project

### PR DESCRIPTION
**Pull request by rutikwankhade**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,9 @@
     <br />
     <a href="https://free-apis.github.io">Free APIs</a> •
     <a href="https://devresourc.es/tools-and-utilities/public-apis">Dev Resources</a> •
-    <a href="https://www.public-apis.ml">Public APIs Site</a>
+    <a href="https://www.public-apis.ml">Public APIs Site</a> •
+    <a href="https://apihouse.vercel.app">Apihouse</a>
+
 </div>
```
Comment (matheusfelipeog) [inline README.md:51]: Remove this spacce, please:

```suggestion
    <a href="https://www.public-apis.ml">Public APIs Site</a> •
    <a href="https://apihouse.vercel.app">Apihouse</a>
</div>
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution and for creating this project, we appreciate it @rutikwankhade 😄 

Can you resolve this review, please?

**Comment by rutikwankhade** [comment]
@matheusfelipeog resolved it. Thank you.

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for the update ;)

**Pull request merged**